addons/analytic/models/analytic_distribution_model.py

```diff
@@ -85,6 +87,8 @@ def _get_applicable_models(self, vals):
         vals = self._get_default_search_domain_vals() | vals
         domain = []
         for fname, value in vals.items():
+            if fname == 'analytic_distribution':
```

Comment:
Instead of doing this, I would extract `analytic_distribution` earlier. It doesn't have any business being here